metsim/metsim.py

```diff
@@ -82,50 +109,57 @@ def write(self, data: dict):
                 'netcdf' : self.write_netcdf,
                 'ascii'  : self.write_ascii
                 }
-        dispatch[self.params.get('out_format', 'netcdf')](data)
+        dispatch[MetSim.params.get('out_format', 'netcdf').lower()](data)
          
 
     def init_netcdf(self):
         """Initialize the output file"""
         print("Initializing netcdf...")
-        out_file = os.path.join(self.params['out_dir'], "forcing.nc")
+        out_file = os.path.join(MetSim.params.get('out_dir', './results/'), "forcing.nc")
```

Comment:
Ooops, this is done in the executable right now, but that would definitely cause issues if someone was trying to script with this.  Fix incoming.